ddocs/medic-client/views/doc_summaries_by_id/map.js

```diff
@@ -1,48 +1,39 @@
 function(doc) {
-  var getPlaceHierarchy = function(contact) {
-    if (!contact) {
-      return;
-    }
-    var place = contact.parent;
+  var getLineage = function(contact) {
     var parts = [];
-    while (place) {
-      if (place.name) {
-        parts.push(place.name);
+    while (contact) {
+      if (contact._id) {
+        parts.push(contact._id);
       }
-      place = place.parent;
+      contact = contact.parent;
     }
     return parts;
   };
 
   if (doc.type === 'data_record' && doc.form) { // report
-    var from = (doc.contact && doc.contact.name) ||
-               doc.from ||
-               doc.sent_by;
-    emit([ doc._id ], {
+    emit(doc._id, {
       _rev: doc._rev,
-      from: from,
+      from: doc.from || doc.sent_by,
       phone: doc.contact && doc.contact.phone,
       form: doc.form,
       read: doc.read,
       valid: !doc.errors || !doc.errors.length,
       verified: doc.verified,
       reported_date: doc.reported_date,
-      place: getPlaceHierarchy(doc.contact)
+      contact: doc.contact && doc.contact._id,
+      lineage: getLineage(doc.contact && doc.contact.parent)
```

Comment:
I don't know about `contactLineage` because it should be consistent with the one below that we use for people types.

I'm also not sure about `lineageIds` because that sets a precedent we're not following anywhere else. Should it also be `fromString` and `readArray` and `formInternalId`?